src/core/shards/core.hpp

```diff
@@ -1965,7 +1993,16 @@ struct Count : SeqUser {
 };
 
 struct Clear : SeqUser {
-  static SHTypesInfo inputTypes() { return CoreInfo::AnyType; }
+  static SHOptionalString help() {
+    return SHCCSTR(
+        "Removes all the elements of the sequence passed to it in the `:Name` parameter. Applicable only to sequences.");
+  }
+
+  static SHTypesInfo inputTypes() { return CoreInfo::NoneType; }
```

Comment:
Why `None`?

Reply:
Reverted this change (rebased and force-pushed to this commit).